internal/tfschema/tfschema.go

```diff
@@ -13,41 +17,58 @@ import (
 	"github.com/env0/terratag/internal/providers"
 	"github.com/env0/terratag/internal/tagging"
 	"github.com/env0/terratag/internal/terraform"
-	"github.com/thoas/go-funk"
 
-	"github.com/hashicorp/go-hclog"
 	"github.com/hashicorp/hcl/v2/hclwrite"
-	"github.com/minamijoyo/tfschema/tfschema"
+	"github.com/zclconf/go-cty/cty"
 )
 
-var providerToClientMap = map[string]tfschema.Client{}
-var providerToClientMapLock sync.Mutex
+var ErrResourceTypeNotFound = errors.New("resource type not found")
 
-var customSupportedProviderNames = [...]string{"google-beta"}
+var providerSchemasMap map[string]*ProviderSchemas = map[string]*ProviderSchemas{}
+var providerSchemasMapLock sync.Mutex
+
+//var customSupportedProviderNames = [...]string{"google-beta"}
+
+type Attribute struct {
```

Comment:
Why do we need to declare those TF-related types? can't we import those types somehow?

Reply:
The external structs are lacking json tagging.
Required structs with json tagging are internal.

E.g.:
https://github.com/hashicorp/terraform/blob/main/internal/command/jsonprovider/provider.go